roles/slave_tools/tasks/nodejs.yml

```diff
@@ -19,6 +19,11 @@
   tags: nodejs
   environment:
     PATH: /opt/build/tools/node-v4.5.0-linux-x64/bin:{{ansible_env.PATH}}
+# Bypass unstable unicode.org
+- file: path=/usr/share/unicode-data state=directory
+  tags: nodejs
+- copy: src=UnicodeData.txt dest=/usr/share/unicode-data/UnicodeData.txt mode=0644 owner=root group=root
+  tags: nodejs
```

Comment:
Why?

Reply:
unicode.org site is very unstable, often not responding. If the file is not available locally, it blocks the slave image build.